### PR TITLE
[ja] Translate /docs/reference/scheduling/policies/ into Japanese

### LINKED ISSUE #32082: [ja] Translate /docs/reference/scheduling/policies/ into Japanese
**This is a Feature Request**

**What would you like to be added**

Translate [reference/scheduling/policies/](https://kubernetes.io/docs/reference/scheduling/policies/) into Japanese.

**Why is this needed**
This page has not been translated.

**Comments**

/language ja
/assign

### PR DESCRIPTION
**Pull request by 196Ikuchil**
Resolves #32082 
Relates PR #32083

### CODE REVIEW

**Review by Arhell — COMMENTED** [review]
/assign @nasa9084

**Comment by sftim** [comment]
(nit)
/retitle [ja] Translate /docs/reference/scheduling/policies/ into Japanese

**Review by nasa9084 — COMMENTED** [review]
/approve
/cc @kakts @bells17

File: content/ja/docs/reference/scheduling/policies.md [hunk 1]
```diff
@@ -0,0 +1,19 @@
+---
+title: スケジューリングポリシー
+content_type: concept
+sitemap:
+  priority: 0.2 # スケジューリングポリシーは廃止されました。
+---
+
+<!-- overview -->
+
+バージョンv1.2以前のKubernetesでは、スケジューリングポリシーを使用して、*predicates*と*priorities*の処理を指定することができました。例えば、`kube-scheduler --policy-config-file <filename>` または `kube-scheduler --policy-configmap <ConfigMap>` を実行すると、スケジューリングポリシーを設定することが可能です。
```
Comment (t-inu) [inline content/ja/docs/reference/scheduling/policies.md:10]:
```suggestion
バージョンv1.23より前のKubernetesでは、スケジューリングポリシーを使用して、*predicates*と*priorities*の処理を指定することができました。例えば、`kube-scheduler --policy-config-file <filename>` または `kube-scheduler --policy-configmap <ConfigMap>` を実行すると、スケジューリングポリシーを設定することが可能です。
```
The original text is below.

> In Kubernetes versions before v1.23

Comment (196Ikuchil) [reply content/ja/docs/reference/scheduling/policies.md]: Thank you for your comment.

**Review by t-inu — COMMENTED** [review]
I reviewed again and added some suggestions.

File: content/ja/docs/reference/scheduling/policies.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+バージョンv1.23より前のKubernetesでは、スケジューリングポリシーを使用して、*predicates*と*priorities*の処理を指定することができました。例えば、`kube-scheduler --policy-config-file <filename>` または `kube-scheduler --policy-configmap <ConfigMap>` を実行すると、スケジューリングポリシーを設定することが可能です。
+
+このスケジューリングポリシーは、バージョンv1.23以降のKubernetesではサポートされていません。関連するフラグである、`policy-config-file`,`policy-configmap`、`policy-configmap-namespace`、`use-legacy-policy-config`も同様にサポートされていません。
+代わりに、[スケジューラー設定](/ja/docs/reference/scheduling/config/)を使用してください。
+
+## {{% heading "whatsnext" %}}
+
+* [スケジューリング](/ja/docs/concepts/scheduling-eviction/kube-scheduler/)について学ぶ
+* [kube-scheduler設定](/ja/docs/reference/scheduling/config/)
```
[the remark this answers is not in the document]
Comment (196Ikuchil) [reply content/ja/docs/reference/scheduling/policies.md]: Oops, Thank you!

**Comment by t-inu** [comment]
Looks good.
But I think #32083 needs to be merged first to be make the link (Scheduler Configuration for ja) accessible.

**Comment by t-inu** [comment]
https://github.com/kubernetes/website/pull/32083 has been merged. Now the link for ja is accessible.
/unhold
/lgtm

**Pull request merged**